dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -1540,12 +1538,11 @@
   },
   "parameters": {
     "Endpoint": {
-      "name": "Endpoint",
+      "name": "endpoint",
       "description": "Supported Cognitive Services endpoint (e.g., https://<resource-name>.api.cognitiveservices.azure.com).",
       "x-ms-parameter-location": "client",
       "required": true,
       "type": "string",
-      "format": "uri",
```

Comment:
This is to make consistent with July preview. Is that OK? @heaths

Reply:
For anything new, I recommend declaring the `uri` format. The others should've been before, but it's consider a breaking change to do it after. Most languages will ignore it anyway and just do a string, but some lanugages - like C# - actually end up using a `Uri` class (correctly). If you don't, we'll have to transform it and we're trying to avoid that. The swaggers should be the main source of truth. They are the spec, afterall. :)

Reply:
@heaths there are so many conflict rules and I'm confused which rule trump which rule... This is existing stuff, not new. In /specification folder, the latest stable version and the latest preview has no format, only /dev folder has format uri. Given we cannot break anything GA, we should fix /dev folder as what I'm doing here - remove format uri, right?